testsuite/tests/conftest.py

```diff
@@ -93,6 +93,12 @@ def label(blame):
     return blame("testrun")
 
 
+@pytest.fixture(scope="module")
+def module_label(label):
```

Comment:
Shouldn't be all labels randomized not only module scope?

Reply:
Well, thats what label fixture is only for session scope. I did not yet add the change to every test that uses it but all the stuff deployed should use either module_label or label